lib/prx_auth/rails/ext/controller.rb

```diff
@@ -8,19 +8,29 @@ class SessionTokenExpiredError < RuntimeError; end
 
       PRX_AUTH_ENV_KEY = 'prx.auth'.freeze
       PRX_JWT_SESSION_KEY = 'prx.auth.jwt'.freeze
+      # subtracted from the JWT ttl
       PRX_JWT_REFRESH_TTL = 300.freeze
       PRX_ACCOUNT_MAPPING_SESSION_KEY = 'prx.auth.account.mapping'.freeze
       PRX_USER_INFO_SESSION_KEY = 'prx.auth.info'.freeze
       PRX_REFRESH_BACK_KEY = 'prx.auth.back'.freeze
 
       def prx_auth_token
-        env_token || session_token
-      rescue SessionTokenExpiredError
-        session.delete(PRX_JWT_SESSION_KEY)
-        session.delete(PRX_ACCOUNT_MAPPING_SESSION_KEY)
-        session.delete(PRX_USER_INFO_SESSION_KEY)
+        set_after_sign_in_path
```

Comment:
So we call this both before every action, and here whenever anyone asks for the `prx_auth_token` - why are both necessary?